fastlane/Fastfile

```diff
@@ -0,0 +1,35 @@
+system("npm install shelljs@0.7.0")
+system("npm install -g cordova")
+system("cordova telemetry on")
+
+$os = "ios,android"
+$jenkins = ENV["JENKINS_URL"]
+unless $jenkins.nil?
+	if $jenkins.include?"Android-Package"
+		$os = "android"
+  elsif $jenkins.include?"iOS-Package"
+		$os = "ios"
+	end
+end
+
+lane :all do
+	system("set -o pipefail && node ../test/test_force.js --os=#$os --apptype=native,native_swift,react_native,hybrid_remote,hybrid_local")
+	system("set -o pipefail && node test_force.js --os=#$os --templaterepouri=https://github.com/forcedotcom/SmartSyncExplorerReactNative#master")
```

Comment:
Right now SmartSyncExplorerReactNative at master uses the unstable SDKs (see https://github.com/forcedotcom/SmartSyncExplorerReactNative/blob/master/package.json#L12)
When we release 5.0, we should change the package.json on master to point to v5.0.0 (and tag it), and have an unstable branch pointing to the unstable SDKs. At that point, this fastlane script should point to unstable (in the unstable branch of SalesforceMobileSDK-Package) and it should point to v5.0.0 on master.